liquidhaskell-boot/src/Language/Haskell/Liquid/GHC/Plugin.hs

```diff
@@ -175,6 +175,10 @@ customDynFlags opts hscEnv = do
          --
          -- https://gitlab.haskell.org/ghc/ghc/-/issues/24386
          `gopt_set` Opt_InsertBreakpoints
+         -- Ignore-interface-pragmas need to be unset to have access to
+         -- the RHS unfoldings in the `Ghc.Var`s which is
+         -- needed as part of the reflection of foreign functions in the logic
+         `gopt_unset` Opt_IgnoreInterfacePragmas
```

Comment:
I've observed that setting flags here can have an effect after the plugin finished running and when compilation continues. Perhaps it is not too difficult to rearrange the setting of dynflags so it is only used for verification.

Reply:
If setting this doesn't cause any issues with tests, then it is better to leave it as is.

Reply:
I'll leave it for now